components/gcds-card/index.json-ld

```diff
@@ -0,0 +1,176 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "GCDS card",
+		"fr": "Carte GCDS"
+	},
+	"description": {
+		"en": "Co-existence pilot of the GCDS card in GCWeb.",
+		"fr": "Pilote de co-existance de la carte GCDS dans GCWeb."
```

Comment:
Suggestion in case people are looking for cartes with an s:

```suggestion
		"fr": "Pilote de co-existance des cartes GCDS dans GCWeb."
```

Reply:
fixed